lib/index.js

```diff
@@ -0,0 +1,69 @@
+import {L2RasterProduct, Status} from '@podaac/swodlr-db-models';
+import * as State from './state.js';
+import * as SDSStatuses from './sds_statuses.js';
+import utils from './utils.js';
+
+const logger = utils.getLogger(import.meta.url);
+const sequelize = utils.sequelize;
+const validateJobset = await utils.loadJsonSchema('jobset');
+
+/**
+ * The main entrypoint into the lambda
+ * @param {object} event the event from SQS
+ */
+export async function lambdaHandler(event) {
+  logger.debug(`Records received: ${event['Records'].length}`);
+
+  let jobs = [];
+  for (const record of event['Records']) {
+    const jobset = JSON.parse(record['body']);
+    if (validateJobset(jobset)) {
+      jobs = jobs.concat(jobset['jobs']);
+    } else {
+      logger.error('Jobset failed to validate');
+      logger.debug(`Failed jobset: ${JSON.stringify(jobset)}`);
+    }
+  }
+
+  for (const job of jobs) {
+    const jobStatus = job['job_status'];
+    if (SDSStatuses.WAITING.has(jobStatus)) {
+      continue;
+    }
+
+    const product = await L2RasterProduct.findByPk(job['product_id']);
+    if (product === null) {
+      logger.error(`Product id not found - ${job['product_id']}`);
+      continue;
+    }
+
+    let state; let reason;
+    if (SDSStatuses.FAIL.has(jobStatus)) {
+      state = State.ERROR;
+      reason = `SDS job failed - please contact support`;
+    } else if (SDSStatuses.SUCCESS.has(jobStatus)) {
+      if (job['stage'] == 'submit_evaluate') {
```

Comment:
> Comparison job['stage'] == 'submit_evaluate' may cause unexpected type coercion 

Did you mean `Comparison job['stage'] === 'submit_evaluate'`

Reply:
Updated in [ef3089c](https://github.com/podaac/swodlr-async-update/pull/1/commits/ef3089ce65c42389e83b329a3e25cbfe1ccc1688)